src/WebJobs.Script.WebHost/Models/ApiModelUtility.cs

```diff
@@ -0,0 +1,83 @@
+﻿// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the MIT License. See License.txt in the project root for license information.
+
+using System;
+using System.Collections.ObjectModel;
+using System.Net.Http;
+using Newtonsoft.Json;
+using Newtonsoft.Json.Converters;
+using Newtonsoft.Json.Linq;
+using Newtonsoft.Json.Serialization;
+
+namespace Microsoft.Azure.WebJobs.Script.WebHost.Models
+{
+    /// <summary>
+    /// A utility class that enables creation of <see cref="ApiModel"/> and <see cref="Link"/>
+    /// instances suitable for consumption over our REST API.
+    /// </summary>
+    public static class ApiModelUtility
+    {
```

Comment:
In general, all the types in the WebHost are public since this isn't a class library. So we're not really concerned with visibility like we are for the core Script class lib